### PR TITLE
Upgrade template dependencies

### PR DESCRIPTION
**Pull request by aahung**
*Issue #, if available:*

*Description of changes:*

update template dependencies to the latest versions.

Right now `tests/integration/build_invoke/build_invoke_base.py` failed because of docker request limit.

I ran it locally and they all passed:

```sh
$ pytest -vvv tests/integration/build_invoke/test_build_invoke.py -n 8

...
[gw7] [ 98%] PASSED tests/integration/build_invoke/test_build_invoke.py::BuildInvoke_java8_cookiecutter_aws_sam_step_functions_sample_app_maven::test_buld_and_invoke <- tests/integration/build_invoke/build_invoke_base.py 
tests/integration/build_invoke/test_build_invoke.py::BuildInvoke_java8_al2_cookiecutter_aws_sam_step_functions_sample_app_maven::test_buld_and_invoke <- tests/integration/build_invoke/build_invoke_base.py 
[gw7] [100%] PASSED tests/integration/build_invoke/test_build_invoke.py::BuildInvoke_java8_al2_cookiecutter_aws_sam_step_functions_sample_app_maven::test_buld_and_invoke <- tests/integration/build_invoke/build_invoke_base.py 

================================================================================================================================== 50 passed, 9 skipped, 6 rerun in 1008.59s (0:16:48) ===================================================================================================================================
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by sriram-mv — CHANGES_REQUESTED** [review]
Have we verified that upgrading these dependencies does not break anything?

File: dotnetcore2.1-image/cookiecutter-aws-sam-hello-dotnet-lambda-image/{{cookiecutter.project_name}}/src/HelloWorld/HelloWorld.csproj [hunk 1]
```diff
@@ -10,9 +10,9 @@
   </PropertyGroup>
```
Comment (c2tarun) [inline dotnetcore2.1-image/cookiecutter-aws-sam-hello-dotnet-lambda-image/{{cookiecutter.project_name}}/src/HelloWorld/HelloWorld.csproj:10]: Why did integ tests fail?

Comment (aahung) [reply dotnetcore2.1-image/cookiecutter-aws-sam-hello-dotnet-lambda-image/{{cookiecutter.project_name}}/src/HelloWorld/HelloWorld.csproj:10]: caused by docker request limit. The tests pass locally on my local machine, check the PR description for detail

**Review by sriram-mv — APPROVED** [review]

File: nodejs12.x/cookiecutter-quick-start-web/{{cookiecutter.project_name}}/package.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
     "version": "0.0.1",
     "private": true,
     "dependencies": {
-        "aws-sdk": "^2.437.0"
+        "aws-sdk": "^2.799.0"
```
Comment (mndeveci) [inline nodejs12.x/cookiecutter-quick-start-web/{{cookiecutter.project_name}}/package.json:7]: 799 as software version 😱

**Review by mndeveci — APPROVED** [review]

File: nodejs12.x/cookiecutter-quick-start-web/{{cookiecutter.project_name}}/package.json (hunk 2 above)
Comment (aahung) [reply nodejs12.x/cookiecutter-quick-start-web/{{cookiecutter.project_name}}/package.json:7]: oh no it is 809 now... let's update it every once for a while

**Pull request merged**